### DISCUSSION TITLE
Time signature change

### DISCUSSION THREAD

**Discussion by Rdeblock** [no-chris/chord-mark#667] [Q&A]
I love the ability to change time signature within a song but it seems that it works sometimes but not always. The following example changes to 2/4 just fine at the beginning of the chorus but refuses to change back to 4/4 at the end. What am I doing wrong?

#c
2/4 C/E F Am G 
_Christ a - _lone, _Corner - _stone 
C/E F Am G
_Weak made _strong in the _Saviour’s _love 
C F Am G % 4/4 C
_Through the _storm, _He is _Lord, _Lord of _all 

Here's how it renders (note: there is no 4/4 before the last C in the last line):
Chorus
|2/4 C/E         |F      |Am        |G        |
       Christ a - lone, Corner - stone
|C/E             |F                   |Am         |G        |
Weak made strong in the Saviour's love
|C                 |F         |Am  |G     |%         |C        |
Through the storm, He is Lord, Lord of all

As an experiment, I tried putting the 4/4 in other locations to see which, if any, would render properly. Anywhere in the first line works. Anywhere in any other line does not work. For grins, I entered the example given in the user guide. It worked fine.

**Comment by no-chris** [comment]
Hello, yes that is actually expected since an inline time signature change is only effective until the end of the current line, or until a new inline change is declared

So it already reverts to 4/4 at the second chord line in your example, and it does not detect any change when you explicitly specify 4/4 on the last line, thus do not render the time signature

I suggest to write your example as follow:
```
#c
2/4
C/E F Am G
_Christ a - _lone, _Corner - _stone
C/E F Am G
_Weak made _strong in the _Saviour’s _love
C F Am G % 4/4 C
_Through the _storm, _He is _Lord, _Lord of _all
```

**Reply by no-chris** [reply]
for more information on inline time signature changes see https://chordmark.netlify.app/docs/reference/chords#time-signature
